modules/installation-configuration-parameters.adoc

```diff
@@ -779,6 +779,14 @@ On clusters that use Kuryr, {rh-openstack} Octavia does not support availability
 |For compute machines, the availability zone to install root volumes on. If you do not set a value for this parameter, the installer selects the default availability zone.
 |A list of strings, for example `["zone-1", "zone-2"]`.
 
+|`compute.platform.openstack.serverGroupPolicy`
+|Server group policy to apply to the group that will contain the compute machines in the pool. You cannot change server group policies or affiliations after creation. Supported options include `anti-affinity`, `soft-affinity`, and `soft-anti-affinity`. The default value is `soft-anti-affinity`. 
+
+An `affinity` policy prevents migrations, and therefore affects {rh-openstack} upgrades. The `affinity` policy is not supported.
+
+If you use a strict `anti-affinity` policy, an additional {rh-openstack} host is required during instance migration. 
```

Comment:
Is the user going to know why? If not, it might be worth clarifying.

Reply:
Yes. If you care about affinity, you should be familiar with that slice of OSP.